Sources/XCTestDynamicOverlay/Internal/Breakpoint.swift

```diff
@@ -1,36 +0,0 @@
-#if canImport(Darwin)
-  import Darwin
-
-  /// Raises a debug breakpoint if a debugger is attached.
-  @inline(__always)
-  @usableFromInline
-  func breakpoint(_ message: @autoclosure () -> String = "") {
```

Comment:
Snuck in a few other small changes, including getting rid of `breakpoint` and instead relying on runtime warnings.